clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects_test.go

```diff
@@ -52,21 +51,21 @@ var (
 	image         = "gcr.io/some-project/some-image"
 )
 
-var node1 = corev1.Node{
```

Comment:
v1 by itself doesn't make much sense unless someone check whats v1 referring. 
I would argue, if we are making this effort we should go in another direction and `'s/v1/corev1/g'`